cli/generic/forms/fields.py

```diff
@@ -36,6 +38,68 @@ def _deserialize(self, value, attr, obj, **kwargs):
         return address
 
 
+class FamilyNameField(fields.Field):
+    """
+    Implements validation of the family name.
+
+    References:
+        - https://marshmallow.readthedocs.io/en/3.0/custom_fields.html
+    """
+
+    def _deserialize(self, value, attr, obj, **kwargs):
+        """
+        Validate data (family name) that was passed to field.
+        """
+        if value not in FAMILY_NAMES:
+            raise ValidationError(f'The following family name `{value}` is invalid.')
+
+        return value
+
+
+class TransactionIdentifiersListField(fields.Field):
+    """
+    Implements validation of the identifier of the list.
+
+    References:
+        - https://marshmallow.readthedocs.io/en/3.0/custom_fields.html
+    """
+
+    def _deserialize(self, value, attr, obj, **kwargs):
+        """
+        Validate data (list identifiers) that was passed to field.
+        """
+        value = value.split(',')
+        list_identifiers = []
```

Comment:
No sense in the variable. Rename to `validated_identifiers`.

Reply:
@dmytrostriletskyi, done.